constraints.txt

```diff
@@ -0,0 +1,83 @@
+# Constrains for testing Diazo / Plone Packages
+# ---------------------------------------------
+
+# tox (Test invocation tool) + virtualenv:
+tox==2.7.0
```

Comment:
This version is no longer the current one. For the others I did not check. I propose we should be able to run on the current releases.